WLogger/WeekLogs/WeekLogsTableViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  WeekLogsTableViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+func mapArray<ArrayValue, NewValue, Error: ErrorType>(producer: SignalProducer<[ArrayValue], Error>,
+    transform: ArrayValue -> NewValue) -> SignalProducer<[NewValue], Error> {
+    return producer.map { $0.map(transform) }
+}
+
+public class WeekLogsTableViewModel {
+    
+    private let _logs = MutableProperty<[LogViewModel]>([])
+    public let fetchLogs: Action<AnyObject, [LogViewModel], RepositoryError>
+    
```

Comment:
add the following public property

```swift
public let logs: AnyProperty<[LogViewModel]>
```

and initialize it like `self.logs = AnyProperty(_logs)`